wot-security-metadata.md

```diff
@@ -978,10 +978,10 @@ Schemes only resolve into specific security mechanisms when combined with specif
 Right now only the following combinations are supported:
 - `basic`+`https`: Basic HTTP Authentication
     - When used in combination with a `proxyUrl` tag indicates Basic HTTP Proxy Authentication
-- `digest`+`http`: Digest HTTP Authentication
+- `digest`+`http`: HTTP Digest Authentication
```

Comment:
probably should review this; using plain HTTP with digest or basic is not a best practice